tests/integration/test_cli_integration.py

```
@@ -0,0 +1,442 @@
+
+    @patch('gh_pr.cli.GitHubClient')
+    def test_review_command(self, mock_client_class):
+        """Test review command."""
+        mock_client = Mock()
+        mock_client_class.return_value = mock_client
+
+        mock_pr = Mock()
+        mock_pr.number = 456
+        mock_pr.create_review = Mock(return_value=Mock(id=1))
+
+        mock_repo.get_pull.return_value = mock_pr
```

Ruff (0.13.1)

Undefined name `mock_repo`

(F821)